src/component/Drobdown.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useCallback, useMemo, useState } from 'react';
+import debounce from 'lodash.debounce';
```

Comment:
![image](https://github.com/mate-academy/react_autocomplete/assets/56752562/1165fbb6-d0e9-4a9c-8a5a-b178719fc5ba)
Try to fix all linter errors